content/concepts/payment-system-basics/accounts/depositauth.md

```diff
@@ -24,12 +26,17 @@ To get the full effect of Deposit Authorization, Ripple recommends also doing th
 
 ## Precise Semantics
 
-An account with Deposit Authorization enabled:x
+An account with Deposit Authorization enabled:
 
-- **Cannot** be the destination of [Payment transactions][], with **one exception**:
+- **Cannot** be the destination of [Payment transactions][], with **the following exceptions**:
     - If the account's XRP balance is equal to or below the minimum account [reserve requirement](reserves.html), it can be the destination of an XRP Payment whose `Amount` is equal or less than the minimum account reserve (currently 20 XRP). This is to prevent an account from becoming "stuck" by being unable to send transactions but also unable to receive XRP. The account's owner reserve does not matter for this case.
-- Can receive XRP from [PaymentChannelClaim transactions][] **only if** the sender of the PaymentChannelClaim transaction is the destination of the payment channel.
-- Can receive XRP from [EscrowFinish transactions][] **only if** the sender of the EscrowFinish transaction is the destination of the escrow.
+    - If the destination has [preauthorized](#preauthorization) the sender of the Payment. _(Requires the [DepositPreauth amendment][])_
```

Comment:
I'm wondering if it makes sense to swap the order of these two bullets.  With some luck preauthorization will be a more common reason for a payment to be accepted.  So it might be nice to list the more common occurrence first.  But that's just a thought.  Use your best judgement.

Reply:
Done in b4ff6b2f493854d7fff6dc2d1fb4b2e659385eb8